src/main/java/io/jenkins/plugins/designlibrary/Inputs.java

```diff
@@ -40,45 +37,29 @@
  */
 @Extension
 public class Inputs extends UISample {
+
+    @NonNull
+    @Override
+    public String getDisplayName() {
+        return super.getDisplayName();
+    }
+
     @Override
     public String getIconFileName() {
         return "symbol-textbox";
     }
 
     @Override
-    public String getDisplayName() {
-        return "Input Fields";
+    public String getDescription() {
+        return "Captures user input through various text or data entry formats.";
     }
 
     public String getShell() {
         return "echo \"Hello World\"";
     }
 
-    public String getHtml() {
-        return "<a href=\"https://jenkins.io\" target=\"_blank\">Jenkins</a>";
-    }
-
     @Extension
     public static final class DescriptorImpl extends UISampleDescriptor {
-        /**
-         * This method provides auto-completion items for the 'state' field.
-         * Stapler finds this method via the naming convention.
-         *
-         * @param value
-         *      The text that the user entered.
-         */
-        public AutoCompletionCandidates doAutoCompleteState(@QueryParameter String value) {
-            AutoCompletionCandidates c = new AutoCompletionCandidates();
-            for (String state : STATES)
-                if (state.toLowerCase().startsWith(value.toLowerCase())) {
-                    c.add(state);
-                }
-            return c;
-        }
-
-        public ComboBoxModel doFillStateItems() {
```

Comment:
this example seems to be missing?

Reply:
Hmm yeah, wasn't 100% on including it, are there any differences apart from it showing all items when focused?

Reply:
yes you can filter it yourself which is really useful.

maybe less so if this ever gets implemented as an includes would cover a lot of cases:
https://open-ui.org/components/combobox.explainer/#introducing-search-attribute

Reply:
ah right yes I got it confused.

The other difference is that it has to be an item in the list.

Take e.g. Azure VM agents, all sizes are known, the user shouldn't be able to enter their own size (using autocomplete right now due to only autocomplete allowing control of filtering though

Reply:
Gotcha, added combobox back. This sound okay?

<img width="913" alt="image" src="https://github.com/user-attachments/assets/79be93d2-733e-48d6-97a1-2cef05ddc13a" />

Reply:
I think so not really sure why autocomplete is on the select page though as its not a select.

Reply:
That's fair - was more of a placeholder for when `select` is updated.